scipy/optimize/_dual_annealing.py

```diff
@@ -448,9 +449,13 @@ def dual_annealing(func, bounds, args=(), maxiter=1000,
         ``f(x, *args)``, where ``x`` is the argument in the form of a 1-D array
         and ``args`` is a  tuple of any additional fixed parameters needed to
         completely specify the function.
-    bounds : sequence, shape (n, 2)
-        Bounds for variables.  ``(min, max)`` pairs for each element in ``x``,
-        defining bounds for the objective function parameter.
+    bounds : sequence or `Bounds`
+        Bounds for variables. There are two ways to specify the bounds:
+        1. Instance of `Bounds` class.
+        2. ``(min, max)`` pairs for each element in ``x``, defining the finite
```

Comment:
Are we happy with these rendering inline or do we want them to be a list? If so, I think you need blank lines before and after (but no indent).

Reply:
TBH, I just copied this from differential_evolution. Visually using a single backtick might be more pleasing though.

Reply:
And I copied the image from there : )

It's not about the backticks. Single and double backticks are used "correctly" here, at least according to numpydoc standards and our Sphinx theme. The question is about whether we want the numbered list items to appear on separate lines.

Reply:
THe style from the `minimize` docstring looks nicer. Will use sth similar unless there are any objections.

![image](https://user-images.githubusercontent.com/40656107/150607073-a6a2a85a-c352-4a6f-bc0f-894733066c08.png)

Reply:
Yes, that's what I'm getting at.
>  If so, I think you need blank lines before and after (but no indent).

Please just don't indent. This is why there is an erroneous vertical, gray bar to the left of the list that is meant to mark block quotes.

Reply:
Adressed. ReST surprises every day :)